mica/stats/acq_stats.py

```diff
@@ -441,6 +448,43 @@ def _get_obsids_to_update(check_missing=False):
     return obsids
 
 
+def warn_on_acq_anom(acq, starcheck, to_addr):
+    BOX_PAD = 16
+    anom_match = (((np.abs(acq['yang_obs'] - acq['yang']) > (acq['halfw'] + BOX_PAD))
```

Comment:
Comment like `# Find slots where the star is acquired outside the search box (with padding)`.